src/metabase/api/search.clj

```diff
@@ -94,7 +96,8 @@
          :search-native-query                 search_native_query
          :search-string                       q
          :table-db-id                         table_db_id
-         :verified                            verified}))))
+         :verified                            verified
+         :ids                                 (set ids)}))))
```

Comment:
It bothers me that `nil` gets turned into `#{}` here - i.e. turns truthy.

```suggestion
         :ids                                 (when (seq ids) (set ids))}))))
```

Reply:
Consistency is why the inconsistent `{:min 1}` goaded me 😄 

Having a `non-empty-set` helper for all these cases could be nice I guess, but don't want to blow up the scope. No objection to leaving as is.